data/mil2525d/utilities/military-features-utilities/README.md

```diff
@@ -18,6 +20,66 @@
 
 ## Instructions
 
+### Create Military Features Geodatabase Template
+
+#### Overview
+
+The utility creates a new File Geodatabase (GDB) in a specified folder, replacing any existing geodatabase of the same name in that folder.  The version of the newly created geodatabase can also be specified.
+
+The structure of the geodatabase is determined by the contents of the CSV files found in a  folder selected at run-time.  The most recent version of these CSV files can be found [here](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/military_feature_schemas).
+
+A Versions table is added to the newly created geodatabase.  The contents of that Versions table is based on the Versions.csv file found in the input folder, [here for example](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/military_feature_schemas/Versions.csv).  The "Version" information for `automated_creation_date` and `last_modification_date` are populated with the current date when the tool is run.
```

Comment:
Solution will be to use the version.csv already in this repo and add two new item value pairs to include the date/time stamp when the script is run.